suisa_sendemeldung/suisa_sendemeldung.py

```diff
@@ -553,46 +293,42 @@ def get_isrc(music: Any) -> str:  # noqa: ANN401
     return isrc
 
 
-def get_csv(data: dict, args: ArgparseNamespace) -> str:
+def get_csv(data: list, settings: Settings) -> str:
     """Create SUISA compatible csv data.
 
     Arguments:
     ---------
         data: To data to create csv from
-        args: Parsed arguments
+        settings: The settings provided to the script
 
     Returns:
     -------
         csv: The converted data
 
     """
-    station_name = args.station_name
+    station_name = settings.station.name
     header = [
-        "Titel",
-        "Komponist",
-        "Interpret",
-        "Interpreten-Info",
         "Sender",
+        "Titel des Musikwerks",
+        "Name des Komponisten",
+        "Interpret(en)",
         "Sendedatum",
         "Sendedauer",
         "Sendezeit",
-        "Werkverzeichnisangaben",
         "ISRC",
         "Label",
-        "CD ID / Katalog-Nummer",
+        "Identifikationsnummer",
+        "Eigenaufnahmen",
+        "EAN/GTIN",
+        "Albumtitel / Titel des Tonträgers",
         "Aufnahmedatum",
         "Aufnahmeland",
-        "Erstveröffentlichungsdatum",
-        "Titel des Tonträgers (Albumtitel)",
-        "Autor Text",
-        "Track Nummer",
-        "Genre",
-        "Programm",
+        "Erstveröffentichungsdatum",
+        "Katalog-Nummer / CD ID",
+        "Werkverzeichnisangaben",
         "Bestellnummer",
-        "Marke",
-        "Label Code",
-        "EAN/GTIN",
-        "Identifikationsnummer",
+        "Veröffentichungsland",
```

Comment:
```suggestion
        "Veröffentlichungsland",
```

Spelling correction.

Reply:
Same here, the template that was ratified by ESchK introduces this error.

Reply:
Indeed, that term is misspelled in the published template as well. IMO it should be corrected, as it is clearly incorrect. If there are any complaints, the typo can be reintroduced, though I highly doubt that will occur. 😉